integration_test/third_party_apps_test/applications/mssql/enable

```diff
@@ -11,6 +11,20 @@ Copy-Item -Path 'C:\Program Files\Google\Cloud Operations\Ops Agent\config\confi
 
 # Configure the Ops Agent.
 Add-Content 'C:\Program Files\Google\Cloud Operations\Ops Agent\config\config.yaml' "
+logging:
+  receivers:
+    mssql_v2:
```

Comment:
the logging receiver `type` in this case would use the name defined in the Type() function - which would be `mssql_errorlog`. We also want to use the default config wherever possible in integration tests, so we wouldn't specify `include_paths` unless they are nonstandard. `receiver_version` is also not relevant to the logging receiver, it's specific to metrics.

The v2 language is because there have been two major iterations of MSSQL metrics support, v2 shouldn't be used in the language for logging.